src/components/DeleteAccountButton/styles.ts

```diff
@@ -0,0 +1,15 @@
+import {StyleSheet} from 'react-native';
+
+const SEIZE = 100;
```

Comment:
```suggestion
const SIZE = 100;
```

I think you have a typo in this name